src/ui/components/Card/index.js

```diff
@@ -8,12 +8,26 @@ export default class Card extends React.Component {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
-    footer: PropTypes.node,
+    footerLink: PropTypes.node,
+    footerText: PropTypes.node,
     header: PropTypes.node,
   }
 
   render() {
-    const { children, className, footer, header } = this.props;
+    const { children, className, footerText, footerLink, header } = this.props;
+
+    let footer;
+    let footerClass;
+    if (footerText && footerLink) {
+      throw new Error(
+        'You cannot specify footerLink and footerText at the same time');
```

Comment:
Yeah, I saw that in the other component but this is a little different because `footerLink` is a node of components. It's not a single `Link`, it's just a node that contains one or more `Links`. There is no way to inject `footerText` into it.